firestore.rules

```diff
@@ -1,11 +1,6 @@
 rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
-    // Start by closing all access by default, then add what you need
-    match /{document=**} {
-      allow read, write: if false;
-    }
-
```

Comment:
As the comment states, you should start by closing all access to the database by default, then add the rules for specific paths. Since you removed this rule, any database operations that you didn't explicitly prohibit in this file will be allowed.